tests/models/test_tpu.py

```diff
@@ -486,3 +486,31 @@ def teardown(self, stage):
 
     model = DebugModel()
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
+
+
+@RunIf(tpu=True)
+@pl_multi_process_test
+def test_tpu_spawn_debug_plugin(tmpdir):
```

Comment:
I think this should be a unit test.

Running 4 epochs on 8 cores seems overkill to check that the string is registered...

Reply:
Tests updated!

Reply:
With unit test I mean, why not test the plugin registry directly?

Or do we not have any other tests checking `tpu_spawn` with `debug=True`?